tests/providers/google/cloud/operators/test_dataproc.py

```diff
@@ -590,7 +681,7 @@ def test_execute(self, mock_hook, mock_uuid):
             query=self.query,
             variables=self.variables,
         )
-        op.execute(context={})
+        op.execute(context=MagicMock())
```

Comment:
I suppose you did this because of adding `xcom_push` in `execute`, right?

Reply:
Yes.

Reply:
Have added a mock task instance for testing the xcom push. This change is regarding the operator extra links. Please take a look at the updated tests.